packages/nmap.vm/tools/chocolateyuninstall.ps1

```diff
@@ -18,7 +18,7 @@ try {
     Uninstall-BinFile -Name "zenmap"
 
     # Silently uninstall
-    VM-Uninstall-With-Uninstaller "Nmap*" "EXE" "/S"
+    VM-Uninstall-With-Uninstaller "Nmap*" $category "EXE" "/S"
```

Comment:
We may also want to try to address this issue noted here: https://github.com/mandiant/VM-Packages/issues/1055#issuecomment-2133920024

It seems like we might want to attempt to run the uninstaller before removing all of the files, if I'm reading it correctly. This would require modifying the `VM-Uninstall-With-Uninstaller` function within `common.vm`.